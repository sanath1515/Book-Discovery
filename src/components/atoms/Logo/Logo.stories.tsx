import React from "react";
import Logo from "./Logo";

export default {
  title: "atoms/Logo",
};

export const Default = (Props) => <Logo />;
