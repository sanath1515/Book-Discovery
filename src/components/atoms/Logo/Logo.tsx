import React from "react";
import { makeStyles } from '@material-ui/core/styles';

export type LogoProps = {}

const useStyles = makeStyles({
  root: {
    width: '35px',
    height: '24px',
    objectFit: "contain",
    margin: '7px 17px 7px 0'
  }
});


const Logo: React.FC<LogoProps> = () => {
  const classes = useStyles();
  return (
    <img className={classes.root} src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/1515BA80-801B-4E2F-9576-01A4265C0E6A.svg" />
  );
}

export default Logo;
