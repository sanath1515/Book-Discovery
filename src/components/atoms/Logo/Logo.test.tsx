import { render } from "@testing-library/react";
import Logo from "./Logo";

describe("logo renders correctly", () => {
  test("should match snapshot", () => {
    const { asFragment } = render(<Logo />);
    expect(asFragment()).toMatchSnapshot();
  });
});
