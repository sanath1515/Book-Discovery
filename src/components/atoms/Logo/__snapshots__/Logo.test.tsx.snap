// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`logo renders correctly should match snapshot 1`] = `
<DocumentFragment>
  <img
    class="makeStyles-root-1"
    src="https://cdn.zeplin.io/5f03041d8c8fc11335546618/assets/1515BA80-801B-4E2F-9576-01A4265C0E6A.svg"
  />
</DocumentFragment>
`;
